### PR TITLE
made smol

### PR DESCRIPTION
**Pull request by Qwerty-Space**

### CODE REVIEW

**Comment by udf** [comment]
thanks fren

**Comment by Qwerty-Space** [comment]
ur welcom bud

File: stdplugins/sp_h.py [hunk 1]
```diff
@@ -1,8 +1,6 @@
 from telethon import events
 from telethon.tl import types
 
-@borg.on(events.NewMessage(pattern=r"^h$", outgoing=False))
+@borg.on(events.NewMessage(pattern="^h$", outgoing=False))
```
Comment (muhammedfurkan) [inline stdplugins/sp_h.py:4]: how to define **@borg** ?

Comment (Qwerty-Space) [reply stdplugins/sp_h.py:4]: with care

Comment (muhammedfurkan) [reply stdplugins/sp_h.py:4]: I dont understand can you explain

Comment (Qwerty-Space) [reply stdplugins/sp_h.py:4]: yes, it was done rather carefully.


care (noun): A concerned or troubled state of mind, as that arising from serious responsibility; worry.

Comment (udf) [reply stdplugins/sp_h.py:4]: @muhammedfurkan Stop being a skiddy and learn to code. It's obviously injected when the plugin is loaded.

**Pull request merged**